packages/react-components/src/theming/themeDuplicates.ts

```diff
@@ -0,0 +1,1167 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { BrandVariants } from '@fluentui/react-components';
+
```

Comment:
Let's have a comment here to explain (link to) where this has come from and why we need to copy from fluent UI

Reply:
Updated